### PR TITLE
Automatically publish of OFEPs as a website

### LINKED ISSUE #76: Publish OFEPs to a website that’s searchable.
```[tasklist]
### Tasks
- [ ] Publish OFEPs to a GitHub page
- [ ] Automatically publish when an OFEP PR has been merged
- [ ] Extend OFEPs to include useful metadata (e.g. status, component, author)
- [ ] Each OFEP can be linked to directly
```

### PR DESCRIPTION
**Pull request by mihirm21**
## This PR

- uses [readthedocs](https://docs.readthedocs.io/en/stable/) and sphinx to upload the ofeps to a website which builds itself automatically after every merge.

### Related Issues

Fixes #76 

### Notes

[Corresponding website](https://ofep-testing.readthedocs.io/)
[Testing repository](https://github.com/mihirm21/ofep-testing)

### Follow-up Tasks

In `docs/source/conf.py`, please change the project information after readthedocs setup with respect to the repository

### CODE REVIEW

File: docs/source/index.rst
Comment (beeme1mr) [inline docs/source/index.rst:1]: Isn't this file generated? Can we gitignore it?

**Review by beeme1mr — COMMENTED** [review]
Can you please document how the publishing process works? It's unclear if the build process needs to be run manually or if the readthedocs handles it for you.

File: docs/source/index.rst
Comment (mihirm21) [reply docs/source/index.rst:1]: Yep, it works without having it initialized.

**Comment by mihirm21** [comment]
On local computer, one could test by just writing a single command, `make html` inside docs directory. Else, for a real website updates, readthedocs does everything for you, from running the scripts to making html page of every `.md` or `.rst` file which resides inside the `docs/source` subdirectory. readthedocs can get triggered on a PR merge.

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+# -- Project information -----------------------------------------------------
+project = 'OFEP-TESTING'
+copyright = '2023, Mihir Mittal'
+author = 'Mihir Mittal'
+release = '1'
```
[the remark this answers is not in the document]
Comment (mihirm21) [reply docs/source/conf.py:15]: Nope, not necessary to update.

**Comment by beeme1mr** [comment]
Closing in favor of #89

**Pull request closed without merging**